content/en/docs/distributions/gke/deploy/overview.md

```diff
@@ -0,0 +1,54 @@
++++
+title = "Overview"
+description = "Full fledged Kubeflow deployment on Google Cloud"
+weight = 1
++++
+
+This guide describes how to deploy Kubeflow and a series of Kubeflow components on GKE (Google Kubernetes Engine). 
+If you want to use Kubeflow Pipelines only, redirect to [Kubeflow Pipelines](/docs/components/pipelines/) for installation
```

Comment:
`redirect to` -> `refer to`
also the link `/docs/components/pipelines/` seems a bit off, I think we can just tell people to look at `/docs/components/pipelines/installation/overview/`, there are already links to installation docs there.

Reply:
Make sense, I remove this line because the next line has the link to installation option.